index.js

```diff
@@ -200,10 +200,7 @@ export default class Supercluster {
                 ]],
                 tags: c.numPoints ? getClusterProperties(c) : this.points[c.index].properties
             };
-            const id = c.numPoints ? c.id : this.points[c.index].id;
-            if (id !== undefined) {
-                f.id = id;
-            }
+            f.id = c.numPoints ? c.id : (this.points[c.index].id || c.index);
```

Comment:
This will break with points that have an id of `0`. Better check for `undefined`.

Reply:
Also, instead of always assigning an `id`, I would put this behind an option (e.g. `generateIds`), like discussed in #97.

Reply:
We relay the options manually in GL JS. Let's go with `generateId` though to be consistent with `geojson-vt` which already has that option.